hr_attendance_rfid/models/hr_employee.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2017 Comunitea Servicios Tecnológicos S.L.
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+from odoo import api, models, fields, _
```

Comment:
@hveficent IMHO uses an alphabetic naming convention, because better to us.

 ex. api, fields, models,  _